pylint/checkers/base.py

```diff
@@ -1003,14 +1009,17 @@ def open(self):
         self.stats = self.linter.add_stats(module=0, function=0, method=0, class_=0)
 
     @utils.check_messages("using-constant-test")
+    @utils.check_messages("missing-parentheses-for-call-in-test")
     def visit_if(self, node):
         self._check_using_constant_test(node, node.test)
 
     @utils.check_messages("using-constant-test")
```

Comment:
`check_messages` accepts a list of messages so this can be squashed into `@check_messages('using-constant-test', 'missing-parantheses-for-call-in-test')```